code/modules/power/supermatter/supermatter.dm

```diff
@@ -628,20 +628,29 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			env.merge(removed)
 			air_update_turf(FALSE, FALSE)
 
-	//Makes em go mad and accumulate rads.
-	var/toAdd = -0.05
+	// Defaults to a value less than 1. Over time the psyCoeff goes to 0 if
+	// no supermatter soothers are nearby.
+	var/psyCoeffDiff = -0.05
 	for(var/mob/living/carbon/human/l in view(src, HALLUCINATION_RANGE(power)))
-		// Someone (generally a Psychologist), when looking at the SM
+		// Someone (generally a Psychologist), when looking at the SMd
```

Comment:
Since you're already refactoring this block of code, can you change `var/mob/living/carbon/human/L` to a more descriptive variable name here?

Reply:
Sure! Why rename SM to SMd though?